content/_ext/changelog_releases.rb

```diff
@@ -21,6 +21,10 @@ def execute(site)
         page.release = changelog_entry
         page.type = @limit_version_key
         page.title = page.title + " for " + changelog_entry.version
+        page.backlink = {
+          label: "Back to changelogs",
```

Comment:
```suggestion
          label: "Back to changelog",
```
Should be singular changelog here.

Reply:
done sir